front/app/containers/Admin/formBuilder/containers/projects/forms/edit/index.tsx

```diff
@@ -115,40 +113,42 @@ export const SurveyEdit = () => {
       h="100vh"
     >
       <FocusOn>
-        <SurveyBuilderTopBar surveyCustomFields={surveyCustomFields} />
+        <FormBuilderTopBar formCustomFields={formCustomFields} />
         <Box mt={`${stylingConsts.menuHeight}px`} display="flex">
-          <SurveyBuilderToolbox onAddField={onAddField} />
+          <FormBuilderToolbox onAddField={onAddField} />
           <StyledRightColumn>
             <Box width="1000px" bgColor="white" minHeight="300px">
-              {!isNilOrError(surveyCustomFields) && (
-                <SurveyFields
+              {!isNilOrError(formCustomFields) && (
+                <FormFields
                   onEditField={setSelectedField}
-                  surveyCustomFields={surveyCustomFields}
+                  formCustomFields={formCustomFields}
                   handleDragRow={handleDragRow}
                   handleDropRow={handleDropRow}
                   selectedFieldId={selectedField?.id}
                 />
               )}
             </Box>
           </StyledRightColumn>
-          <SurveyBuilderSettings
-            key={selectedField ? selectedField.id : 'no-field-selected'}
-            field={selectedField}
-            onDelete={handleDelete}
-            onFieldChange={onFieldChange}
-            onClose={closeSettings}
-          />
+          {!isNilOrError(selectedField) && (
+            <FormBuilderSettings
+              key={selectedField ? selectedField.id : 'no-field-selected'}
```

Comment:
We could set this to `selectedField.id` since selectedField will always be there here

Reply:
Ah true! Ok, I'll update it. Thanks!